generate-graphql-docs/config.yml

```diff
@@ -0,0 +1,10 @@
+info:
```

Comment:
@quesabe can you please share a link which explains the format and purpose of the file? Couldn’t find it after quick googling.